DIRECTORY.md

```diff
@@ -1,4 +1,4 @@
-# List of all files
+#numberi List of all files
```

Comment:
Why change?

---
**Sema Reaction:** :question: I have a question

Reply:
thanks for the correction
i changed my last commit